dispatcher/backend/src/routes/languages/language.py

```diff
@@ -0,0 +1,127 @@
+import json
+from http import HTTPStatus
+
+import trafaret as t
+from flask import request, Response, jsonify
+
+from common.mongo import Languages
+from errors.http import InvalidRequestJSON
+from routes import authenticate, errors
+from routes.base import BaseRoute
+from routes.languages.base import LanguageQueryMixin
+
+
+class LanguagesRoute(BaseRoute):
+    rule = '/'
+    name = 'languages'
+    methods = ['POST', 'GET']
+
+    @authenticate
+    def post(self, *args, **kwargs):
```

Comment:
Sorry I think language should be a read only thing. Its value should come from the schedules. I will explain more in the main comment of the review.